app/ui-react/syndesis/src/modules/data/shared/ViewSqlFormAndTable.tsx

```diff
@@ -0,0 +1,122 @@
+import { QueryResults } from '@syndesis/models';
+import { Container, GenericTable } from '@syndesis/ui';
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
I dont this tslint line should be needed?

Reply:
see below